packages/dbml-connector/src/connectors/mssqlConnector.ts

```diff
@@ -229,7 +261,7 @@ const generateTablesFieldsAndEnums = async (client: sql.ConnectionPool, schemas:
       constraints_with_row_number
     WHERE
       rn = 1
-      ${buildSchemaQuery('table_schema', schemas, 'WHERE')}
+      ${buildSchemaQuery('table_schema', schemas)}
```

Comment:
Why do you need to change this em? I believe the old one is correct 🤔

Reply:
No anh, the previous caused error since the generated query looks like this:
```
 WHERE
      rn = 1
      WHERE table_schema IN ('dbo')
```

Reply:
aw, i see. Ok em